index.html

```diff
@@ -7,93 +7,211 @@
-                        <h4>Frameworks</h4>
-                    <ul>
-                        <li>Flask</li>
-                        <li>Ruby on Rails</li>
-                        <li>SQlite</li>
-                        <li>Bootstrap</li>
-                    </ul>
-                    </div>
+    <div class="main-container flex column">
+        <div class="sub-container-1 flex column">
+            <header class="toolbar flex">
+                <a class="logo" href="_blank"> Mo&apos;athal S. Kachi</a>
+                <nav class="button nav"></nav>
+            </header>
+
+            <section class="headline flex column">
+                <div class="head flex column">
+                    <h1 class="mainTxt">Mo&apos;athal</h1>
+
+                    <h3 class="spprtTxt">Software Developer</h3>
+                </div>
 
-                    <div class="frame">
-                        <h4>Skills</h4>
-                        <ul>
-                        <li>Codekit</li>
-                        <li>Github</li>
-                        <li>Gitlab</li>
-                        <li>Codepen</li>
-                        <li>Terminal</li>
-                    </ul>
+                <div class="spprtSect flex column">
+                    <p class="offer">
+                        I can help you build a product , feature or website Look through some of my work and experience!
+                        If
+                        you like what you see and have a project you need coded, don’t hesitate to contact me.
+                    </p>
+
+                    <hr class="divider">
+
+                    <div class="sect flex">
+                        <div class="frame">
+
+                            <h4>Languages</h4>
+                            <ul>
+                                <li>SQL</li>
+                                <li>HTML</li>
+                                <li>CSS</li>
+                                <li>Javascript</li>
+                                <li>Ruby</li>
+                                <li>SQL</li>
+                            </ul>
+                        </div>
+
+                        <div class="frame">
+                            <h4>Frameworks</h4>
+                            <ul>
+                                <li>Flask</li>
+                                <li>Ruby on Rails</li>
+                                <li>SQlite</li>
+                                <li>Bootstrap</li>
+                            </ul>
+                        </div>
+
+                        <div class="frame">
+                            <h4>Skills</h4>
+                            <ul>
+                                <li>Codekit</li>
+                                <li>Github</li>
+                                <li>Gitlab</li>
+                                <li>Codepen</li>
+                                <li>Terminal</li>
+                            </ul>
+                        </div>
```

Comment:
- [ ] This section should be wrapped with a `section` tag for semantic reasons, kindly change that. 😁😁